packages/material-ui/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -60,6 +105,18 @@ const Breadcrumbs = React.forwardRef(function Breadcrumbs(props, ref) {
     ...other
   } = props;
 
+  const styleProps = {
```

Comment:
Let's move these after the line 120 (where we are defining the `expanded` state) and add it also in the `styleProps`